src/test/resources/com/puppycrawl/tools/checkstyle/checks/todocomment/InputTodoCommentDefault.java

```diff
@@ -0,0 +1,15 @@
+/*
+TodoComment
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.todocomment;
+
+public class InputTodoCommentDefault {
+    int i;
+    public void method() {
+        i++; // TODO: do differently in future // violation 'Comment matches to-do format 'TODO:''
```

Comment:
As we targeting comment, please move `// violation ....` on line above or below and skip usage of TODO in message.
We should not mix them and we should see clearly what triggers violation.

Please keep this in mind for all comment and javadoc Checks .

Reply:
Done

Reply:
![Screenshot from 2023-07-10 11-46-18](https://github.com/checkstyle/checkstyle/assets/97679350/c71a7b78-af5c-42ad-a049-53848e30d308)
I am not sure why this is happening.

Reply:
may be https://github.com/checkstyle/checkstyle/blob/0831953d82df93e5ebe7f178c2f2b904af8b8f3b/src/main/java/com/puppycrawl/tools/checkstyle/checks/TodoCommentCheck.java#L133 
this needs to be TokenTypes.SingleLineComment and TokenTypes.BlockComment 

I have worked with comment in fall-thru pr but i never face this in that pr